.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   "extends": "airbnb",
   "rules": {
+    "linebreak-style": 0,
```

Comment:
Can we delete this line? We don't want to have multiple different line endings in our code. Just force users to use LF. They can configure their environment to use those.